### PR TITLE
Pbc/carrier management version

### PR DESCRIPTION
**Pull request by andriitserkovnyi**
## PR Description

Add versioning
Add installation section

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/pbc/all/carrier-management/202204.0/import-and-export-data/file-details-shipment-price.csv.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ data:import:shipment-price
 
 | PARAMETER | REQUIRED | TYPE | REQUIREMENTS OR COMMENTS | DESCRIPTION |
 | --- | --- | --- | --- | --- |
-| shipment_method_key | &check; | String  | Value previously imported already by its data importer using the [shipment.csv](/docs/pbc/all/carrier-management/import-and-export-data/file-details-shipment.csv.html) file.| Identifier of the shipment method. |
+| shipment_method_key | &check; | String  | Value previously imported already by its data importer using the [shipment.csv](/docs/pbc/all/carrier-management/{{site.version}}/import-and-export-data/file-details-shipment.csv.html) file.| Identifier of the shipment method. |
```
[the remark this answers is not in the document]
Comment (andriitserkovnyi) [reply docs/pbc/all/carrier-management/202204.0/import-and-export-data/file-details-shipment-price.csv.md:29]: No in this one

File: docs/pbc/all/price-management/install-and-upgrade/upgrade-modules/upgrade-to-multi-currency.md [hunk 2]
```diff
@@ -76,7 +76,7 @@ composer update spryker/*
 
 4. For Shipment, we have changed the way the default price is assigned (currency-aware). Zed money input form has also been changed to accept multi-currency:
 
-   * **Shipment >= 6.** — see [Migration Guide - Shipment](/docs/pbc/all/carrier-management/install-and-upgrade/upgrade-the-shipment-module.html).
+   * **Shipment >= 6.** — see [Migration Guide - Shipment](/docs/pbc/all/carrier-management/{{site.version}}/install-and-upgrade/upgrade-the-shipment-module.html).
```
Comment (VadymSachenko) [inline docs/pbc/all/price-management/install-and-upgrade/upgrade-modules/upgrade-to-multi-currency.md:79]: spaces should be removed
```suggestion
   * **Shipment >= 6.**—see [Migration Guide - Shipment](/docs/pbc/all/carrier-management/{{site.version}}/install-and-upgrade/upgrade-the-shipment-module.html).
```

File: docs/pbc/all/price-management/install-and-upgrade/upgrade-modules/upgrade-to-multi-currency.md [hunk 3]
```diff
@@ -76,7 +76,7 @@ composer update spryker/*
...
    * **ShipmentCartConnector** - is a new module, which provides plugins for cart to handle cases when currency changed. <!-- add a link See Integration guide for more details.-->
```
Comment (VadymSachenko) [inline docs/pbc/all/price-management/install-and-upgrade/upgrade-modules/upgrade-to-multi-currency.md:80]: same
```suggestion
   * **ShipmentCartConnector**—is a new module, which provides plugins for cart to handle cases when currency changed. <!-- add a link See Integration guide for more details.-->
```

**Review by VadymSachenko — COMMENTED** [review]
See my comments.

**Pull request merged**